### PR TITLE
snark transaction profiler job in CI

### LINKED ISSUE #11600: Transaction snark profiler in CI

### PR DESCRIPTION
**Pull request by dkijania**
Explain your changes:
New job for running snark transaction profiler

Explain how you tested your changes:

Run CI

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #11600

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by deepthiskumar — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

File: buildkite/scripts/run-snark-transaction-profiler.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+set -eo pipefail
+
+# Don't prompt for answers during apt-get install
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update
+apt-get install -y git apt-transport-https ca-certificates tzdata curl python3
+
+TESTNET_NAME="berkeley"
```
[the remark this answers is not in the document]
Comment (dkijania) [reply buildkite/scripts/run-snark-transaction-profiler.sh:11]: Right. I will add case for TESTNET_NAME

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Pull request merged**